packages/core/src/common/_variables.scss

```diff
@@ -4,6 +4,9 @@
 @import "colors";
 @import "mixins";
 
+$pt-namespace: "pt" !default;
```

Comment:
why not one var like `$namespace: "pt" !default` or `$blueprint-namespace: "pt" !default`?